api/controllers/builds.go

```diff
@@ -21,7 +22,21 @@ import (
 func BuildList(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 	app := mux.Vars(r)["app"]
 
-	builds, err := provider.BuildList(app)
+	l := r.URL.Query().Get("limit")
+
+	var err error
+	var limit int
+
+	if l == "" {
```

Comment:
Yeah I would like an idomatic way to do pagination, limits, filters with a QueryOptions type but this was just a pragmatic solution